web/src/main/java/org/cbioportal/web/StaticDataTimestampController.java

```diff
@@ -0,0 +1,35 @@
+package org.cbioportal.web;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.cbioportal.model.TableTimestampPair;
+import org.cbioportal.service.StaticDataTimestampService;
+import org.cbioportal.web.config.annotation.InternalApi;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collections;
+import java.util.List;
+
+@InternalApi
+@RestController
+@Validated
+@Api(tags = "Timestamps", description = " ")
+public class StaticDataTimestampController {
+    private static final List<String> tables = Collections.singletonList("gene");
```

Comment:
Whats cached on the frontend?  Should you add reference_genome_gene table here?  That contains chr, cytoband, and some other props.

Reply:
Oh, you're right. Looking at the mapper, the fields come entirely from `reference_genome_gene`, but there is an inner join with `gene` on `entrez_gene_id`. I think the answer is that the cache should be invalidated if either change, so I'll include both tables. 

The frontend caches the full superagent response object that it makes when it calls `/api/reference-genome-genes/hg19`